Create U-22反省.md

nakamuratomohiko opened the pull request:
反省点を書いたページ

nakamuratomohiko committed:
Create U-22反省.md

反省点を書いたページ

nakamuratomohiko commented:
日本語のおかしいところがないかをチェックして取り込んでくれ

wamekukyouzin reviewed (changes requested):
+ 箇条書きにおいて末尾に句点は必要ない。また、直後に丸カッコが来る場合、丸カッコの後に配置する
+ 「考慮していないことで」は、より因果関係のわかりやすい「考慮していなかったため」の方がよい
+ 「進まない状態が生じた」より「進まなかった」のほうがよい
+ 「頼れなかった」は口語的なので、「人に頼ることができなかった」等に書き換える

nakamuratomohiko committed:
Update U-22反省.md

句点の削除
以下の置換
「考慮していないことで」 ->「考慮していなかったため」
「進まない状態が生じた 」-> 「進まなかった」
「頼れなかった」 -> 「人に頼ることができなかった」

nakamuratomohiko commented:
指摘された箇所を変更した
